### ISSUE TITLE
Warnings when trying to build a project into exe with pywin .

### ISSUE THREAD

**Issue by SHAANBHAYA** [pyinstaller/pyinstaller#5138]
I saw that when building a file with the pywintypes imported the crt dll that are to be linked are sometime not due to this error.

```
4203 WARNING: Can not get binary dependencies for file: C:\Program Files (x86)\Windows Kits\10\Redist\10\ucrt\DLLs\x64\api-ms-win-core-processthreads-l1-1-0.dll
4203 WARNING:   Reason: too many values to unpack (expected 2)
Traceback (most recent call last):
  File "d:\work\merge\ddclient\venv\lib\site-packages\PyInstaller\depend\bindepend.py", line 731, in getImports
    return _getImports_pe(pth)
  File "d:\work\merge\ddclient\venv\lib\site-packages\PyInstaller\depend\bindepend.py", line 128, in _getImports_pe
    dll,_ = forwarder.split('.')
ValueError: too many values to unpack (expected 2)
```

## Context information (for bug reports)

* Output of `pyinstaller --version`:  For version 3.5 till 4.0
* Version of Python: 3.7
* Platform:  Windows 10 
* Did you also try this on another platform? Does it work there? : N/A as this is windows specific library .

### A minimal example program which shows the error
Steps to reproduce 
1. install the following libs : "pypiwin32==223","pywin32==227"
2. Install the CRT for windows 10 from https://www.microsoft.com/en-us/download/details.aspx?id=48234
3. Now create the following script

```
import servicemanager
import sys
import win32event
import win32service
import win32serviceutil
print("Hi")
```
4.Run the pyinstaller --onefile <scriptname>.py

### Stacktrace / full error message
```
 running Analysis Analysis-00.toc
3668 INFO: Adding Microsoft.Windows.Common-Controls to dependent assemblies of final executable
  required by d:\work\merge\ddclient\venv\scripts\python.exe
4113 WARNING: Can not get binary dependencies for file: C:\Program Files (x86)\Windows Kits\10\Windows Performance Toolkit\api-ms-win-core-libraryloader-l1-1-0.dll
4113 WARNING:   Reason: too many values to unpack (expected 2)
Traceback (most recent call last):
  File "d:\work\merge\ddclient\venv\lib\site-packages\PyInstaller\depend\bindepend.py", line 731, in getImports
    return _getImports_pe(pth)
  File "d:\work\merge\ddclient\venv\lib\site-packages\PyInstaller\depend\bindepend.py", line 128, in _getImports_pe
    dll,_ = forwarder.split('.')
ValueError: too many values to unpack (expected 2)
4121 WARNING: Can not get binary dependencies for file: C:\Program Files (x86)\Windows Kits\10\Windows Performance Toolkit\api-ms-win-core-memory-l1-1-0.dll
4122 WARNING:   Reason: too many values to unpack (expected 2)
Traceback (most recent call last):
  File "d:\work\merge\ddclient\venv\lib\site-packages\PyInstaller\depend\bindepend.py", line 731, in getImports
    return _getImports_pe(pth)
  File "d:\work\merge\ddclient\venv\lib\site-packages\PyInstaller\depend\bindepend.py", line 128, in _getImports_pe
    dll,_ = forwarder.split('.')
ValueError: too many values to unpack (expected 2)
4131 WARNING: Can not get binary dependencies for file: C:\Program Files (x86)\Windows Kits\10\Windows Performance Toolkit\api-ms-win-core-processthreads-l1-1-0.dll
4132 WARNING:   Reason: too many values to unpack (expected 2)
Traceback (most recent call last):
  File "d:\work\merge\ddclient\venv\lib\site-packages\PyInstaller\depend\bindepend.py", line 731, in getImports
    return _getImports_pe(pth)
  File "d:\work\merge\ddclient\venv\lib\site-packages\PyInstaller\depend\bindepend.py", line 128, in _getImports_pe
    dll,_ = forwarder.split('.')
ValueError: too many values to unpack (expected 2)
4145 WARNING: Can not get binary dependencies for file: C:\Program Files (x86)\Windows Kits\10\Windows Performance Toolkit\api-ms-win-core-localization-l1-2-0.dll
4146 WARNING:   Reason: too many values to unpack (expected 2)
Traceback (most recent call last):
  File "d:\work\merge\ddclient\venv\lib\site-packages\PyInstaller\depend\bindepend.py", line 731, in getImports
    return _getImports_pe(pth)
  File "d:\work\merge\ddclient\venv\lib\site-packages\PyInstaller\depend\bindepend.py", line 128, in _getImports_pe
    dll,_ = forwarder.split('.')
ValueError: too many values to unpack (expected 2)
4191 WARNING: Can not get binary dependencies for file: C:\Program Files (x86)\Windows Kits\10\Windows Performance Toolkit\api-ms-win-core-namedpipe-l1-1-0.dll
4191 WARNING:   Reason: too many values to unpack (expected 2)
Traceback (most recent call last):
  File "d:\work\merge\ddclient\venv\lib\site-packages\PyInstaller\depend\bindepend.py", line 731, in getImports
    return _getImports_pe(pth)
  File "d:\work\merge\ddclient\venv\lib\site-packages\PyInstaller\depend\bindepend.py", line 128, in _getImports_pe
    dll,_ = forwarder.split('.')
ValueError: too many values to unpack (expected 2)
4204 WARNING: Can not get binary dependencies for file: C:\Program Files (x86)\Windows Kits\10\Windows Performance Toolkit\api-ms-win-core-timezone-l1-1-0.dll
4205 WARNING:   Reason: too many values to unpack (expected 2)
Traceback (most recent call last):
  File "d:\work\merge\ddclient\venv\lib\site-packages\PyInstaller\depend\bindepend.py", line 731, in getImports
    return _getImports_pe(pth)
  File "d:\work\merge\ddclient\venv\lib\site-packages\PyInstaller\depend\bindepend.py", line 128, in _getImports_pe
    dll,_ = forwarder.split('.')
ValueError: too many values to unpack (expected 2)
4221 INFO: Analyzing D:\Work\merge\DDClient\DDClient\dsad.py

```

Note : After going through the code flow it seems that the problem lies at  : PyInstaller/depend/bindepend.py:128
```

def _getImports_pe(pth):
    """
    Find the binary dependencies of PTH.

    This implementation walks through the PE header
    and uses library pefile for that and supports
    32/64bit Windows
    """
    dlls = set()
    # By default library pefile parses all PE information.
    # We are only interested in the list of dependent dlls.
    # Performance is improved by reading only needed information.
    # https://code.google.com/p/pefile/wiki/UsageExamples

    pe = pefile.PE(pth, fast_load=True)
    pe.parse_data_directories(directories=[
        pefile.DIRECTORY_ENTRY['IMAGE_DIRECTORY_ENTRY_IMPORT'],
        pefile.DIRECTORY_ENTRY['IMAGE_DIRECTORY_ENTRY_EXPORT'],
        ],
        forwarded_exports_only=True,
        import_dllnames_only=True,
        )

    # Some libraries have no other binary dependencies. Use empty list
    # in that case. Otherwise pefile would return None.
    # e.g. C:\windows\system32\kernel32.dll on Wine
    for entry in getattr(pe, 'DIRECTORY_ENTRY_IMPORT', []):
        dll_str = winutils.convert_dll_name_to_str(entry.dll)
        dlls.add(dll_str)

    # We must also read the exports table to find forwarded symbols:
    # http://blogs.msdn.com/b/oldnewthing/archive/2006/07/19/671238.aspx
    exportSymbols = getattr(pe, 'DIRECTORY_ENTRY_EXPORT', None)
    if exportSymbols:
        for sym in exportSymbols.symbols:
            if sym.forwarder is not None:
                # sym.forwarder is a bytes object. Convert it to a string.
                forwarder = winutils.convert_dll_name_to_str(sym.forwarder)
                # sym.forwarder is for example 'KERNEL32.EnterCriticalSection'
                dll,_ = forwarder.split('.') <------------------------------------------------Problem lies here.
                dlls.add(dll + ".dll")

    pe.close()
    return dlls
```
The assumption is that the variable forwarder will have a name like 'KERNEL32.EnterCriticalSection' with a single dot.But many dlls have names with multiple dots like kernel32.dll.CreateFileMapping or advapi32.dll.EnumDynamicTimeZoneInformation . Due to this the error of **too many values to unpack happens**.
A better solution will be to change the line to this
```
dll= forwarder.split('.')[0] 
```

**Comment by bwoodsend** [comment]
@SHAANBHAYA Good catch. Could you please create a pull request with your proposed change, referencing this issue.

**Comment by SHAANBHAYA** [comment]
Sure will do that .

**Comment by rokm** [comment]
This was fixed by #5068.
